### PR TITLE
add antd v1.1.0 with npm autoupdate config

### LINKED ISSUE #7886: [Request] Add antd
**Library name:** antd
**Git repository url:** https://github.com/ant-design/ant-design
**npm package url(optional):** https://www.npmjs.com/package/antd
**License(s):** MIT LICENSE
**Official homepage:** http://ant.design/
**Wanna say something? Leave message here:** 

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7886 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ant-design/ant-design
 * Official website (optional, not the repository): http://ant.design
 * NPM package url (optional): https://www.npmjs.com/package/antd
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3901
   - Count of watchers: 319
   - Count of forks: 664
 * NPM download stats (optional):
   - Downloads in the last day: 465
   - Downloads in the last week: 4208
   - Downloads in the last month: 23747

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7886, cc @afc163

### CODE REVIEW

**Comment by afc163** [comment]
:+1: Great job!

File: ajax/libs/antd/package.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+  "name": "antd",
+  "filename": "antd.min.js",
+  "version": "1.1.0",
+  "description": "An UI design language",
+  "homepage": "http://ant.design/",
+  "author": {
+    "name": "偏右",
```
Comment (x09326) [inline ajax/libs/antd/package.json]: I'm not sure whether it can use chinese.
ping @PeterDaveHello

Comment (PeterDaveHello) [inline ajax/libs/antd/package.json]: Does the author has English name? If not, we just use Chinese.

Comment (afc163) [inline ajax/libs/antd/package.json]: afc163

Comment (pvnr0082t) [inline ajax/libs/antd/package.json]: @afc163 Thanks.

Comment (PeterDaveHello) [inline ajax/libs/antd/package.json]: Thanks @afc163

**Comment by pvnr0082t** [comment]
@x09326 I have updated author info. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
Looks like you forgot to rebase your branch with the latest master branch.

**Comment by pvnr0082t** [comment]
@x09326 I have rebase on the latest master branch. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t you should update to the latest version next time.

**Pull request merged**